src/agent/onefuzz/src/asan.rs

```diff
@@ -226,15 +240,28 @@ mod tests {
                 "breakpoint",
                 43,
             ),
+            (
+                "data/asan-check-failure.txt",
+                "AddressSanitizer",
+                "CHECK failed",
+                12,
+            ),
+            (
+                "data/asan-check-failure-missing-symbolizer.txt",
+                "AddressSanitizer",
+                "CHECK failed",
```

Comment:
Double-checking: this test case just tests that if the symbolizer is missing, our parser doesn't choke on that in the output, and can still extract that the problem was a `CHECK failed`, right?

Reply:
Correct.